chainer_chemistry/dataset/parsers/sdf_file_parser.py

```diff
@@ -53,12 +55,19 @@ def parse(self, filepath, return_smiles=False):
         if isinstance(pp, MolPreprocessor):
             mol_supplier = Chem.SDMolSupplier(filepath)
 
+            if target_index is None:
+                target_index = numpy.arange(len(mol_supplier))
```

Comment:
If `mol_supplier` does not accept `numpy.integer`, I think we do not have to prepare `target_index` as a `numpy.ndarray` here.

Reply:
I changed this place.

But I'd like to keep 
```
# `mol_supplier` does not accept numpy.integer, we must use int
mol = mol_supplier[int(index)]
```